### ISSUE TITLE
Marketplace for extensions is empty

### ISSUE THREAD

**Issue by NormanDunbar** [VSCodium/vscodium#142]
When I try to search or install any extensions, then nothing is listed. For example, search in extensions marketplace for "@popular c++" and after a short delay, with the progress indicator working, nothing is listed. "@popular" on it's own simply shows the progress indicator for as long as you leave it.

I don't use VSCode and this is my first experience with VSCodium after a reccommendation in Linux Format magazine. I cannot confirm if this happens in VSCode as well.

- [*] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there. However #107 is _similar_ but not quite the same, I do not get any confirmation messages that "nothing was found" etc.

Steps to reproduce the behaviour:
1. Go to main editor.
2. Click on Extensions button/icon on the left sidebar.
3. Attempt a search.
4. Nothing happens, no extensions are found.

**Expected behavior**
A list of extensions to appear when searching is complete.

**Desktop (please complete the following information):**
 - OS: Linux Mint
 - Architecture: x64
 - Version: 19.1 Tessa

**Additional context**
I looked in `$HOME/.vscode-oss/extensions` - there's absolutely nothing there, no hidden files either.

**Comment by stripedpajamas** [comment]
Hi @NormanDunbar 👋 

Sorry for the rough start to your first time using it VSCode/VSCodium 😢 

I haven't used the `@popular` search term before, but if you just search for `c++` do you not get any results? 

Additionally, can you provide what version of VSCodium you have installed and how you installed it?

**Comment by NormanDunbar** [comment]
Good Evening @stripedpajamas,

No apology necessary, these things happen.

The `@popular` is filled in when I select the popular extensions from the more actions menu button. I get no results when searching for anything I'm afraid. Sometimes the progress indicator stops, other times it keeps going until I clear the search, by clicking that button.

I installed from the instructions on the home page at https://vscodium.com/#install following the Debian instructions. Linuc Mint is Debian based via Ubuntu.

I'm not at my laptop at the moment so I can't say which version, sorry, I should have checked earlier, my bad. :-( 

It will be something simple that's causing the problem I expect.


Cheers,
Norm.

**Comment by NormanDunbar** [comment]
Hi @stripedpajamas,

sorry about the delay, the VSCodium version details are:

````
Version: 1.33.1
Commit: 51b0b28134d51361cf996d2f0a1c698247aeabd8
Date: 2019-04-11T17:47:02.611Z
Electron: 3.1.6
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Linux x64 4.15.0-48-generic
````

I don't know if it makes any difference, but when I first start VSCodium, the extensions icon on the left sidebar is just that "square" thing. As soon as I have a problem with searching, it gets a little blue clock face added - showing a time of 4 o'clock. That stays there until I close down and restart.

Cheers,
Norm.

**Comment by stripedpajamas** [comment]
Hmm, I've never seen it just spin like that. Do you have anything that would block outbound internet connections from applications? 

Otherwise, the only next troubleshooting step would be to install VSCode and see if the issue happens there -- if it does we aren't dealing with a VSCodium-specific issue and can get Microsoft's help in troubleshooting.

**Comment by NormanDunbar** [comment]
Good Evening @stripedpajamas.

I don't have anything that would block my outbound internet connections, no. This problem is on my laptop - there's no firewall getting in the way. I'll try wiping it and reinstalling, then I'll try VSCode and see what happens.

Cheers,
Norm.

**Comment by NormanDunbar** [comment]
Evening @pavanagrawal123,

Yes, I'm sure. I installed from the instructions on the home page at https://vscodium.com/#install following the Debian instructions. Linux Mint is Debian based via Ubuntu.

Cheers,
Norm.

**Comment by NormanDunbar** [comment]
Morning all.

Good news, sort of. 

I had an update of VSCodium this morning, the version number stayed at 1.33.1 though. That didn't sort my marketplace problem.

I wiped and reinstalled - still the same.

I then wiped and installed VSCode following this page's instructions:

https://computingforgeeks.com/install-visual-studio-code-on-ubuntu-18-04-linux-mint-19/

And that installed this version:

````
Version: 1.33.1
Commit: 51b0b28134d51361cf996d2f0a1c698247aeabd8
Date: 2019-04-11T08:20:22.771Z
Electron: 3.1.6
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Linux x64 4.15.0-48-generic
````

And I have exactly the same problem with the Microsoft version - so it's not a VSCodium problem it would seem.

Cheers,
Norm.

**Comment by NormanDunbar** [comment]
@stripedpajamas - We have a solution!

This is weird. After installing VSCode, I noticed my desktop was running a couple of instances of "conky" and when I checked there were actually 4 instances running. When I killed one of them, Marketplace appeared in VSCode instantly.

I wiped VSCode and re-installed VSCodium again. And with one instance of Conky running, I can see and install extensions from Marketplace and also install extensions from the CTRL-P command line.

I have no idea why Conky running 4 instances would kill VSCode/VSCodium but it seems that it does. I'm a happy user now - thanks for the help and apologies for the signal to noise ratio! I just need to find out why Conky is running so many instances.

Cheers,
Norm.
